js/src/get-started-page/get-started-card/index.scss

```diff
@@ -22,11 +22,11 @@
 		display: flex;
 		flex-direction: column;
 		align-items: flex-start;
-		gap: 20px;
+		gap: calc(#{$grid-unit-50} / 2);
```

Comment:
💅 A primitive sass variable can be calcauled directly.
```suggestion
		gap: $grid-unit-50 / 2;
```

Reply:
This creates a warning in the css linter

Reply:
Got it. Looks like it could still be simplified a little bit with `gap: calc($grid-unit-50 / 2);`.

Reply:
![Uploading Screenshot 2022-07-14 at 16.44.20.png…]()

Reply:
Not sure why but I didn't get that warning.

#### Change and rebuild

https://user-images.githubusercontent.com/17420811/179137245-573654a7-ef8e-4a5c-8a23-063ce4c59c9a.mp4

#### `npm start`

![image](https://user-images.githubusercontent.com/17420811/179137525-1e311d83-b148-468e-8ec6-d8e645ef20d7.png)

Reply:
> Not sure why but I didn't get that warning.

The warning is when doing `gap: $grid-unit / 2` if there is inside a calc it works. Seems like `/` is also being used as a separator.

More info:  https://sass-lang.com/documentation/breaking-changes/slash-div

Btw. I added the change removing `#{}` in this commit https://github.com/woocommerce/google-listings-and-ads/pull/1578/commits/1ed4470056ffcf3759ac1993670f5a7e85ed01dc

Reply:
Thanks for the adjustments.

> The warning is when doing `gap: $grid-unit / 2` if there is inside a calc it works. Seems like `/` is also being used as a separator.

I thought https://github.com/woocommerce/google-listings-and-ads/pull/1578#discussion_r921112923 was replying to  https://github.com/woocommerce/google-listings-and-ads/pull/1578#discussion_r921037788.